js-exercises/aperture/aperture.js

```diff
@@ -0,0 +1,16 @@
+function aperture(tupleLength, array) {
+  const result = [];
+  for (let i = 0; i < array.length; i++) {
+    let j = 0;
+    const tupleArray = [];
+    while (j < tupleLength) {
```

Comment:
You do not need to use while loop inside the for loop. You can do something like this:
for (let i = 0; i < (array.length - (tupleLength - 1)); i++) {
    tupleArray.push(array.slice(i, i + tupleLength ));
  }